src/twisted/internet/interfaces.py

```diff
@@ -173,7 +173,7 @@ def resolveHostName(resolutionReceiver, hostName, portNumber=0,
             practice, this means an iterable containing
             L{twisted.internet.address.IPv4Address},
             L{twisted.internet.address.IPv6Address}, both, or neither.
-        @type addressTypes: L{collections.Iterable} of L{type}
+        @type addressTypes: L{collections.abc.Iterable} of L{type}
```

Comment:
The [documentation build](https://buildbot.twistedmatrix.com/builders/documentation/builds/3792) passes because this is pulled from the Python 3 documentation's inter-Sphinx inventory.